Specs/Scene/ModelExperimental/GeometryPipelineStageSpec.js

```diff
@@ -68,6 +68,8 @@ describe(
       "./Data/Models/GltfLoader/BuildingsMetadata/glTF/buildings-metadata.gltf";
     const dracoMilkTruck =
       "./Data/Models/DracoCompression/CesiumMilkTruck/CesiumMilkTruck.gltf";
+    const dracoWaterBottle =
+      "./Data/Models/GltfLoader/WaterBottle/glTF-Draco/WaterBottle.gltf";
```

Comment:
Is there a smaller model that we can use? This one has some pretty large textures.

It looks like the glTF exporter in Blender has an option to output tangents, so maybe you could export a simple box and then Draco compress it with gltf-pipeline.